apps/full-site-editing/full-site-editing-plugin/blog-posts-block/newspack-homepage-articles/blocks/homepage-articles/edit.js

```diff
@@ -201,7 +201,7 @@ class Edit extends Component {
 
 	formatByline = authorInfo => (
 		<span className="byline">
-			{ __( 'by', 'newspack-blocks' ) }{ ' ' }
+			{ __( 'by', 'full-site-editing' ) }{ ' ' }
```

Comment:
I don't think I could translate this to Finnish as-is since we don't have the word "by" in our language. :-/ I'd definitely need context to turn it into a word like "Author:", "Created:", etc.

Reply:
This should be addressed in Newspack code 👍